### ISSUE TITLE
Crash when scrolling a parchment horizontal scrollView inside a android native listview

### ISSUE THREAD

**Issue by mrhether** [EmirWeb/parchment#17]
Here is the xml:

```
`<mobi.parchment.widget.adapterview.listview.ListView
    android:id="@+id/horizontal_list"
    android:layout_width="match_parent"
    android:layout_height="100dp"
    android:background="@android:color/holo_blue_bright"
    parchment:isViewPager="false"
    parchment:orientation="horizontal">`
```

Here is the crashlog:
`09-04 14:43:37.094  16131-16131/com.***.******* E/AndroidRuntime﹕ FATAL EXCEPTION: main
    java.lang.NullPointerException
            at mobi.parchment.widget.adapterview.AdapterViewManager.recycle(AdapterViewManager.java:31)
            at mobi.parchment.widget.adapterview.LayoutManager.layoutCells(LayoutManager.java:592)
            at mobi.parchment.widget.adapterview.LayoutManager.layout(LayoutManager.java:221)
            at mobi.parchment.widget.adapterview.AbstractAdapterView.onLayout(AbstractAdapterView.java:174)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1670)
            at android.widget.LinearLayout.layoutHorizontal(LinearLayout.java:1659)
            at android.widget.LinearLayout.onLayout(LinearLayout.java:1443)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.ListView.setupChild(ListView.java:1921)
            at android.widget.ListView.makeAndAddView(ListView.java:1824)
            at android.widget.ListView.fillDown(ListView.java:674)
            at android.widget.ListView.fillSpecific(ListView.java:1332)
            at android.widget.ListView.layoutChildren(ListView.java:1663)
            at android.widget.AbsListView.onLayout(AbsListView.java:2045)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:446)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:446)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:446)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.support.v4.widget.DrawerLayout.onLayout(DrawerLayout.java:743)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:446)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:446)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:446)
            at android.view.View.layout(View.java:14255)
            at android.view.ViewGroup.layout(ViewGroup.java:4413)
            at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:1998)
            at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1812)
            at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1050)
            at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:4560)
            at android.view.Choreographer$CallbackRecord.run(Choreographer.java:749)
            at android.view.Choreographer.doCallbacks(Choreographer.java:562)
            at android.view.Choreographer.doFrame(Choreographer.java:532)
            at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:735)
            at android.os.Handler.handleCallback(Handler.java:725)
            at android.os.Handler.dispatchMessage(Handler.java:92)
            at android.os.Looper.loop(Looper.java:137)
            at android.app.ActivityThread.main(ActivityThread.java:5171)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:511)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:797)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:564)
            at dalvik.system.NativeStart.main(Native Method)`

**Comment by emir-hasanbegovic** [comment]
Hi @mrhether , I've fixed the issue you reported in version 1.6.9 let me know how that works out for you.

FYI, I did some tests as I've never thought of putting the library into a ListView. I noticed one thing, when scrolling up, ListView doesn't pass proper values for the top and hence, we get items that are laid out off screen. Let me know if you come across this issue and I'll look into it further.
